.github/workflows/scraper.yml

```
@@ -0,0 +1,54 @@
+name: Scraper Test CI
+on:
+  push:
+    paths:
+      - 'pipeline/**'
+      - 'client.py'
+      - '.github/workflows/scraper.yml'
+  pull_request:
+    paths:
+      - 'pipeline/**'
+      - 'client.py'
+      - '.github/workflows/scraper.yml'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports: ['5432:5432']
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+      
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install psycopg2
+          python -m pip install requests
+          python -m pip install datetime
+        if: steps.cache.outputs.cache-hit != 'true'
+
+      - name: Python run script
+        run: python client.py
+        env:
+          POSTGRES_USER: ${{ secrets.DB_USER }}
+          POSTGRES_PASSWORD: ${{ secrets.DB_PASSWORD }}
+          POSTGRES_DB: ${{ secrets.DB_NAME }}
+          POSTGRES_HOST: ${{ secrets.DB_HOST }}
```

Shouldn't these be the hardcoded credentials you set above?
```suggestion
          POSTGRES_USER: postgres
          POSTGRES_PASSWORD: postgres
          POSTGRES_DB: postgres
          POSTGRES_HOST: localhost
```

---

Actually, it doesn't look like you're using these environment variables anywhere else, so I suppose they could be deleted?

---

Yes, you can delete them right now. I had problems to import secrets so just hardcode the credentials into `client.py`.